### PR TITLE
Updating the question form UI to be much simpler

### PR DESCRIPTION
**Pull request by sewagodimo**
@pjdelport I started this PR because I wanted to improve the way we ask questions. At the moment the idea of a heading and a title are  a bit too confusing for some of the students. We have often assumed that they would be asking complex questions but the questions on the site http://edu-buza.herokuapp.com/questions/ show that the students aren't initially going to ask for complex questions. The questions style on Quora is much simpler and I think we should rather aim for that

- [x] Remove the question heading
- [ ] Restyle the question form

### CODE REVIEW

**Comment by PiDelport** [comment]
This sounds like a good simplification, yes!

File: src/buza/templates/buza/question_form.html [hunk 1]
```diff
@@ -14,6 +14,28 @@
...
+      </select>
+  </div>
+    <div class="form-group col-md-6">
+      <label for="inputGrade">Grade</label>
+      <select id="inputGrade" class="form-control">
+        <option>10</option>
+      </select>
+  </div>
+              </div>
+</form>
```
Comment (PiDelport) [inline src/buza/templates/buza/question_form.html:38]: So just to make sure I understand, the layout above is the suggested alternative to the previous / default layout below?

**Review by PiDelport — CHANGES_REQUESTED** [review]
We'll also probably want to:

- [ ] Update the model to remove the heading too
- [ ] Use crispy-forms for the new layout

We can tackle that tomorrow at Codebridge? :)

File: src/buza/templates/buza/question_form.html (hunk 1 above)
Comment (sewagodimo) [reply src/buza/templates/buza/question_form.html:38]: yes

**Comment by sewagodimo** [comment]
@pjdelport I am closing this because a question will just be a textfield

**Comment by PiDelport** [comment]
Cool! Yeah, we have a much better aim now, I think. :)

**Pull request closed without merging**